### ISSUE TITLE
Feature Request: Add-Ins for quick access to functions

### ISSUE THREAD

**Issue by rsangole** [KentonWhite/ProjectTemplate#226]
#### Report an Issue / Request a Feature

I'm submitting a (Check one with "x") : 
- [ ] bug report 
- [x] feature request

***

##### Issue Severity Classification - 

(Check one with "x") : 
- [ ] 1 - Severe
- [ ] 2 - Moderate
- [x] 3 - Low

##### Expected Behavior

I would like some of the common functions to be added into an add-in. Some examples of excellent add-ins which add value from other packages include:

![__documents_onedrive_mspa_453_text_analytics_nasa_text_r_project_-_master_-_rstudio](https://user-images.githubusercontent.com/6344050/37436830-4f5a2c76-27c0-11e8-816a-dc289544daf3.jpg)

I would love to see a button like 'Load Project' which would run:

    library(ProjectTemplate)
    load.project()

which are two commands I frequently type after clearing my environment to ensure full end-to-end reproducibility of my code.

I'm sure other regular users of the package can think of other useful buttons too.

**Comment by Hugovdberg** [comment]
I guess you are suggesting an addin for RStudio? The [documentation](https://rstudio.github.io/rstudioaddins/) looks like this should be not too hard to implement.

I would add two addins:
 - 'Load project': call `require(ProjectTemplate); load.project()`
 - 'Reload project': call `require(ProjectTemplate); reload.project()`

**Comment by rsangole** [comment]
Yep, exactly right. I'll work on this over the next week or so and see if I can get a PR out.

**Comment by rsangole** [comment]
Other users of this package - what other add-ins would you like to see?

**Comment by rsangole** [comment]
Pull request #229 addresses this issue.
